Library/test-helpers/lib.sh

```diff
@@ -1906,62 +1906,21 @@ Returns 0.
 
 =cut
 
-limeconPrepareAgentImage() {
+limeconPrepareImage() {
 
     if [ "$1" == "-f" -o "$1" == "--file" ]; then
```

Comment:
Add check ensuring that exactly two arguments (dockerfile and tag) are given, fail otherwise. Then use `DOCKERFILE=$1` and `TAG=$2`